src/Utilities/constants.js

```diff
@@ -23,6 +23,7 @@ export const INVENTORY_TOTAL_FETCH_URL_SERVER = '/api/inventory/v1/hosts';
 export const INVENTORY_TOTAL_FETCH_EDGE_PARAMS =
   '?filter[system_profile][host_type]=edge&page=1&per_page=1';
 export const INVENTORY_TOTAL_FETCH_CONVENTIONAL_PARAMS = '?page=1&per_page=1';
+export const EDGE_API = '/api/edge/v1';
```

Comment:
I believe there is no need to duplicate the value here: just reuse one we have defined at https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/src/api/api.js#L3.

Reply:
done